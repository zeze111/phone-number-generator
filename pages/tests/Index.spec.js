import React from "react";
import { shallow } from "enzyme";
import Index from "../Index";

describe("Index page", () => {
  it("should render the component", () => {
    const wrapper = shallow(<Index />);
    expect(wrapper).toMatchSnapshot();
  });
});
