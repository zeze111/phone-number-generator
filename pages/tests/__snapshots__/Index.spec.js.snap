// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Index page should render the component 1`] = `
<div>
  <Header />
  <Content />
</div>
`;
